shared/constants/types/fs.js

```diff
@@ -3,23 +3,35 @@ import * as I from 'immutable'
 
 export opaque type Path = ?string
 
-export type PathType = 'folder' | 'file' | 'symlink' | 'exec' | 'unknown'
+export type PathType = 'folder' | 'file' | 'symlink' | 'unknown'
+
+type pendingPathMetadataProgressType = 1
```

Comment:
can you explain what you're trying to do here

Reply:
It's an enum constant.

Reply:
Oh, and specifically: progress is `pending` when we don't have metadata loaded, and `loaded` once we do. Later we might have extra progress values for data being loaded.

Reply:
just use a string so you can tell what it is. with flow there's no value in using an int and a constant all over the place